recipes/cuda-pathfinder/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: cuda-pathfinder
+  version: "1.0.0"
+  # no-arch packages can't use selectors, so we avoid using the "match(python, "<3.9")" selector this way
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  # TODO: switch to use the stable release url in the next release
+  url: https://github.com/nvidia/cuda-python/archive/${{ name }}-v${{ version }}.tar.gz
+  sha256: 98820d855f06ec23828669196d2d21e310095809dd89ba6d32b875637a1d9656
+
+build:
+  noarch: python
+  script: cd cuda_pathfinder && python -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - cuda-version >=11.0
+
+tests:
+  - python:
+      imports:
+      - cuda.pathfinder
+      pip_check: true
+      python_version: ${{ python_min }}.*
+
+about:
+  # TODO: update the url once the docs is up
+  homepage: https://nvidia.github.io/cuda-python/latest/
+  summary: 'cuda.pathfinder: Utilities for locating CUDA components in a Python virtual environment'
+  description: |
+    This is a placeholder.
```

Comment:
xref https://github.com/NVIDIA/cuda-python/issues/761, whatever we write there can be reused here, and vice versa